arviz/rcparams.py

```diff
@@ -93,11 +102,60 @@ def _validate_boolean(value):
     return value is True or value == "true"
 
 
+def make_iterable_validator(scalar_validator, length=None):
+    """Validate value is an iterable datatype."""
+    # based on matplotlib's _listify_validator function
+
+    def validate_iterable(value):
+        if isinstance(value, str):
+            val = tuple([scalar_validator(v.strip("([ ])")) for v in value.split(",") if v.strip()])
+            if length is not None:
+                if len(val) == length:
+                    return val
+                else:
+                    raise ValueError("Iterable must be of length: {}".format(length))
+            else:
+                return val
+        elif np.iterable(value) and not isinstance(value, (set, frozenset)):
+            val = tuple([scalar_validator(v) for v in value])
+            if length is not None:
+                if len(val) == length:
+                    return val
+                else:
+                    raise ValueError("Iterable must be of length: {}".format(length))
+            else:
+                return val
+        else:
+            raise ValueError("Only iterable values are valid")
+
+    return validate_iterable
+
+
+def _make_validate_float_iterable(length):
+    """Validate value is a float iterable of given length."""
+    # uses make_iterable_validator to validate float iterables
+
+    def validate_float_iterable(value):
+        if value is None or isinstance(value, str) and value.lower() == "none":
+            return None
+        elif value == "auto":
+            return value
+        try:
+            validate_floatlist = make_iterable_validator(_validate_float_or_none, length)
+            return validate_floatlist(value)
+        except:
+            raise ValueError("Only float iterable values are valid.")
+
+    return validate_float_iterable
```

Comment:
Using the code above we could define here:

    _validate_bokeh_bounds = make_iterable_validator(
        _validate_float_or_none, length=2, allow_none=True, allow_auto=True
    )